### ISSUE TITLE
运行问题

### ISSUE THREAD

**Issue by jirouhanjinlun** [octree-nn/octfusion#6]
我在配置好环境后使用sh scripts/run_snet_cond.sh generate hr chair指令，得到如下结果

continue training with ckpt=saved_ckpt/diffusion-ckpt/im_5/df_steps-union.pth
[*] Training is starting on jiroujinlun, GPU#: 0, logs_dir: logs
[*] Training with command: 
CUDA_VISIBLE_DEVICES=0 python3 train.py --name chair_union/union_2t_test_lr2e-4 --logs_dir logs --gpu_ids 0 --mode generate     --lr 2e-4 --epochs 500  --min_lr 
1e-6 --warmup_epochs 40 --update_learning_rate 0 --ema_rate 0.999 --seed 42     --model union_2t --stage_flag hr --df_cfg configs/octfusion_snet_cond.yaml --ckpt
_num 3 --category chair     --vq_model GraphVAE --vq_cfg configs/vae_snet_eval.yaml --vq_ckpt saved_ckpt/vae-ckpt/vae-shapenet-depth-8.pth     --display_freq 1000 --print_freq 25     --save_steps_freq 3000 --save_latest_freq 500     --debug 0 --ckpt saved_ckpt/diffusion-ckpt/im_5/df_steps-union.pth
请问为什么会出现这种情况？我应该如何解决？

**Comment by wst2001** [comment]
您好！可以粘贴一下具体的报错信息吗？目前上面的都是正常的一些输出

**Comment by jirouhanjinlun** [comment]
> 您好！可以粘贴一下具体的报错信息吗？目前上面的都是正常的一些输出

您好，这就是我所有的输出内容了。我之后会尝试您给我的单gpu运行的代码修改建议，然后再看看是否还会有异常情况出现。非常感谢您的耐心回复！
